server/src/main/java/org/elasticsearch/transport/SniffConnectionStrategy.java

```diff
@@ -107,13 +115,13 @@ public class SniffConnectionStrategy extends RemoteConnectionStrategy {
             Setting.Property.NodeScope
         )
     );
-
     static final int CHANNELS_PER_CONNECTION = 6;
 
     private static final Predicate<DiscoveryNode> DEFAULT_NODE_PREDICATE = (node) -> Version.CURRENT.isCompatible(node.getVersion())
         && (node.isMasterNode() == false || node.canContainData() || node.isIngestNode());
 
     private final List<String> configuredSeedNodes;
+    private final String configuredAuthorization;
```

Comment:
Perhaps just `authorization`, the `configured` prefix is there for seed nodes to distinguish from the resolved seed nodes, but here we only need the one field I think.

Reply:
I changed it from `configuredAuthorization` to authorization. I was following the `configuredSeeds` example, but it is OK to just call it `authorization`.